tapir/coop/tests/test_create_membershipresignation.py

```diff
@@ -15,27 +16,17 @@ def test_membershipResignation_created_isTrue(self):
         resigned_member = MembershipResignationFactory.create()
 
         response = self.client.post(
-            reverse(
-                "coop/resign_member/new",
-                {
-                    "share_owner": resigned_member,
-                    "cancellation_date": datetime.now(),
-                    "pay_out_day": datetime.now() + datetime(day=31, month=12, years=3),
-                    "cancellation_reason": resigned_member.cancellation_reason,
-                    "resignation_type": resigned_member.resignation_type,
-                    "transferring_shares_to": resigned_member.transferring_shares_to,
-                    "paid_out": resigned_member.paid_out,
-                },
-                follow=True,
-            )
+            reverse("coop:resign_new_membership"),
+            {
+                "share_owner": resigned_member,
```

Comment:
That won't work since `resigned_member` is not a ShareOwner, it's a MembershipResignation.